frameworks/helloworld/src/main/dist/pod-profile-mount-volume.yml

```diff
@@ -28,7 +35,14 @@ pods:
       noprofile:
         goal: RUNNING
         essential: false
-        cmd: df -t $VOLUME_PROFILE pod-container-path >> pod-container-path/output && df -t ext4 noprofile-container-path >> noprofile-container-path/output && sleep $SLEEP_DURATION
+        cmd: >
+               set -e -x
+               [ -n "{{{TEST_PROFILE_VOLUME_COMMAND}}}" ]
```

Comment:
Same question here? Should this be an `if` statement with the code below included in the body?

Reply:
Just saw your comment above.